lib/modules/messages-submit-widgets/index.js

```diff
@@ -0,0 +1,38 @@
+var sendemail = require('sendemail');
+var sendMany = sendemail.sendMany;
+
+module.exports = {
+  extend: "apostrophe-pieces-submit-widgets",
+  fields: ["subject", "details", "title", "email"],
+  construct: function(self, options) {
+    self.beforeInsert = function(req, piece, callback) {
+      var person = {
+        toName: 'admin',
+        email: process.env.SENDER_EMAIL_ADDRESS,
+        subject: piece.subject,
+        details: piece.details,
+        clientEmail: piece.email,
+        clientName: piece.title
+      };
+      var detailsForSendMany = {
+        templateName: 'template',
+        context: person,
+        subject: person.subject,
+        toAddresses: [person.email],
+        senderEmailAddress: person.senderEmailAddress,
+        replyToAddress: person.replyToAddress,
+        ccAddresses: null,
+        bccAddresses: null
```

Comment:
If we get this `sendemail` pr merged: https://github.com/dwyl/sendemail/pull/87, then we can clean up some of this code

Reply:
I think that would be best. I have reviewed [dwyl/sendemail#87](https://github.com/dwyl/sendemail/pull/87) now. Do you want to hold off on merging this until that has been merged or shall we merge this and refactor it later?

Reply:
I think we should merge this pr in, then we can make the `sendMany -> email` change later

Reply:
Sounds good